mars/services/task/execution/ray/executor.py

```diff
@@ -258,6 +286,10 @@ async def execute_subtask_graph(
             elif output_count == 1:
                 output_object_refs = [output_object_refs]
             self._cur_stage_output_object_refs.extend(output_object_refs)
+            # Note: `output_object_refs` is greater than 0, if not it won't run
```

Comment:
Perhaps `output_count` will never be 0. If not, please remove thsoe comments

Reply:
As far as i know, it would not be 0. I can remove the comments.